src/Pscid/Options.purs

```diff
@@ -121,18 +156,19 @@ buildOptions
   → Eff (fs ∷ FS | e) PscidOptions
 buildOptions port testAfterRebuild includes outputDirectory censor = do
   defaults ← unwrap <$> mkDefaultOptions
-  let sourceDirectories =
-        if null includes
-        then defaults.sourceDirectories
-        else filter (not null) (split (Pattern ";") includes)
+  let includesArr = filter (not null) (split (Pattern ";") includes)
+      sourceDirectories = defaults.sourceDirectories <> includesArr
       censorCodes = filter (not null) (split (Pattern ",") censor)
+      buildCommand = setCommandIncludes includesArr defaults.buildCommand
+      testCommand = setCommandIncludes includesArr defaults.testCommand
+
   pure (wrap { port: fromNumber (readInt 10 port)
              , testAfterRebuild
              , sourceDirectories
              , censorCodes
-             , buildCommand: defaults.buildCommand
+             , buildCommand: buildCommand
              , outputDirectory
-             , testCommand: defaults.testCommand
+             , testCommand: testCommand
```

Comment:
Another opportunity for a record pun, like above

Reply:
Done.